src/radar/radar/radar_controller.py

```diff
@@ -0,0 +1,123 @@
+import cantools
+import rclpy
+from opencaret_msgs.msg import CanMessage, RadarTrack, RadarTrackAccel, RadarTracks
+from rclpy.node import Node
+from util import util
+import os.path
+import opendbc
+
+class ECU:
+    CAM = 0  # camera
+    DSU = 1  # driving support unit
+    APGS = 2  # advanced parking guidance system
+
+
+class CAR:
+    PRIUS = 0
+    LEXUS_RXH = 1
+    RAV4 = 2
+    RAV4H = 3
+    COROLLA = 4
+
+
+STATIC_MSGS = [
+    (0x141, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 1, 2, '\x00\x00\x00\x46'),
+    (0x128, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 1, 3,
+     '\xf4\x01\x90\x83\x00\x37'),
+    (0x283, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 0, 3,
+     '\x00\x00\x00\x00\x00\x00\x8c'),
+    (0x344, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 0, 5,
+     '\x00\x00\x01\x00\x00\x00\x00\x50'),
+    (0x160, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 1, 7,
+     '\x00\x00\x08\x12\x01\x31\x9c\x51'),
+    (0x161, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 1, 7,
+     '\x00\x1e\x00\x00\x00\x80\x07'),
+    (0x365, ECU.DSU, (CAR.RAV4, CAR.COROLLA), 0, 20, '\x00\x00\x00\x80\xfc\x00\x08'),
+    (0x366, ECU.DSU, (CAR.RAV4, CAR.COROLLA), 0, 20, '\x00\x72\x07\xff\x09\xfe\x00'),
+    (0x4CB, ECU.DSU, (CAR.PRIUS, CAR.RAV4H, CAR.LEXUS_RXH, CAR.RAV4, CAR.COROLLA), 0, 100,
+     '\x0c\x00\x00\x00\x00\x00\x00\x00'),
+]
+
+
+class RadarController(Node):
```

Comment:
Would be good to change this class name to something like ToyotaRadarController since it only works with Toyota radar units.